src/Common/Services/Mail/MailConfiguration.cs

```diff
@@ -4,6 +4,12 @@ namespace Passwordless.Common.Services.Mail;
 
 public class MailConfiguration
 {
+    [Obsolete]
+    public string? From { get; set; }
+
+    [Obsolete]
+    public string? FromName { get; set; }
+
```

Comment:
Couldn't we keep this a non-obsolete and just use them as the default for every provider? Only requiring users to start using channels if they want to separate different emails

Reply:
Makes sense, but we won't have any references to it in the rest of the code. It will just create a default channel if they do not exist.